api/src/tests/unit/data_providers/repositories/test_TodoRepository.py

```diff
@@ -12,52 +12,52 @@ def _setup_repository(self, test_client: MongoDatabaseClient):
         self.repository = TodoRepository(client=test_client)
 
     def test_create(self):
-        todo_item = TodoItem(id="1234", title="todo 1")
+        todo_item = TodoItem(id="1234", title="todos 1")
         self.repository.create(todo_item)
         assert len(self.repository.get_all()) == 1
 
     def test_create_already_exists(self):
-        todo_item_1 = TodoItem(id="1234", title="todo 1")
+        todo_item_1 = TodoItem(id="1234", title="todos 1")
         self.repository.create(todo_item_1)
         with pytest.raises(ValidationException):
-            todo_item_2 = TodoItem(id="1234", title="todo 1")
+            todo_item_2 = TodoItem(id="1234", title="todos 1")
             self.repository.create(todo_item_2)
 
     def test_find_item_that_exist(self):
         documents = [
-            {"_id": "81549300", "title": "todo 1"},
-            {"_id": "1a2b", "title": "todo 2"},
-            {"_id": "987321", "title": "todo 3"},
+            {"_id": "81549300", "title": "todos 1"},
+            {"_id": "1a2b", "title": "todos 2"},
+            {"_id": "987321", "title": "todos 3"},
             {
                 "_id": "987456",
-                "title": "todo 4",
+                "title": "todos 4",
             },
         ]
         self.repository.client.insert_many(documents)
-        todo_item = self.repository.find_one({"title": "todo 2"})
+        todo_item = self.repository.find_one({"title": "todos 2"})
         assert todo_item.id == "1a2b"
```

Comment:
Surely a single todo should be named `todo [id]`? Why all the `todos` when talking about a single todo?

Reply:
Agreed